snowpack/src/build/optimize.ts

```diff
@@ -51,7 +51,7 @@ const FAKE_BUILD_DIRECTORY_REGEX = /.*\~\~bundle\~\~[\\\/]/;
  * Scan a directory and remove any empty folders, recursively.
  */
 async function removeEmptyFolders(directoryLoc: string): Promise<boolean> {
-  if ((await fs.stat(directoryLoc)).isDirectory()) {
+  if (!(await fs.stat(directoryLoc)).isDirectory()) {
```

Comment:
we only want to run this function if it is a directory. the code below (line 57) seems to be the code that you want to fix (detects if the folder actually is empty or not).

Reply:
Perhaps this behaviour is a bit different on a linux system, but when I was doing some bruteforce debugging I noticed that it never goes beyond that line. It detects `directoryLoc` is indeed a folder using `isDirectory()` then skips it by going to the next line `return false`, leaving the directory and everything in it intact. This simply leaves a huge amount of empty folders in my situation.

Reply:
just redid a test, and it pretty much just stops at root output dir
```
[snowpack] ! building source files...
[snowpack] ✔ build complete [2.88s]
[snowpack] ! installing dependencies...
[snowpack] ✔ install complete! [5.92s]
[snowpack] ! verifying build...
[snowpack] ✔ verification complete [0.06s]
[snowpack] ! writing build to disk...
[snowpack] ! optimizing build...
[optimize] (early preview: experiments.optimize is experimental, and still subject to change.)
[debug] removeEmptyFolders - C:\_Projects\xxxxxxxx\public\dist - true - (skipped)
[snowpack] ✔ optimize complete [0.79s]
[snowpack] ▶ Build Complete!
```

Reply:
oh my god, of course! Not only did I write it wrong, but I mixed the two up when I read this PR! Great catch :)